### PR TITLE
readme: Add content menu and code quality badge

### PR DESCRIPTION
**Pull request by netoax**
What does this implement/fix? Explain your changes
---------------------------------------------------
This PR submit minor changes on the README to improve documentation and to add the codacy code quality badge.

Does this close any currently open issues?
------------------------------------------
Nops.

**Operating System/Platform:** macOS 10.14 - Darwin 18.0.0

**Go Version:** go1.12.7 darwin/amd64

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,19 @@
...
 
-## Installation and usage
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/9140aa8c06934071ad6e3cf3b1b148ff)](https://www.codacy.com/manual/joaoaneto/knot-babeltower?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=CESARBR/knot-babeltower&amp;utm_campaign=Badge_Grade)
+
+## Contents
+ - [Basic installation and usage](#basic-installation-and-usage)
+    - [Requirements](#requirements)
+    - [Configuration](#configuration)
+    - [Setup](#setup)
+    - [Compiling and Running](#compiling-and-running)
```
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

**Comment by joaoaneto** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 20
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4651373&bid=15577405)

File: README.md [hunk 2]
```diff
@@ -1,6 +1,19 @@
...
+ - [Docker installation and usage](#docker-installation-and-usage)
+    - [Requirements](#requirements)
+    - [Building and Running](#building-and-running)
```
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 1 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md [hunk 3]
```diff
@@ -1,6 +1,19 @@
...
+ - [Verify service health](#verify-service-health)
```
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md [hunk 4]
```diff
@@ -1,6 +1,19 @@
...
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/9140aa8c06934071ad6e3cf3b1b148ff)](https://www.codacy.com/manual/joaoaneto/knot-babeltower?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=CESARBR/knot-babeltower&amp;utm_campaign=Badge_Grade)
+
+## Contents
+ - [Basic installation and usage](#basic-installation-and-usage)
+    - [Requirements](#requirements)
+    - [Configuration](#configuration)
+    - [Setup](#setup)
+    - [Compiling and Running](#compiling-and-running)
+ - [Docker installation and usage](#docker-installation-and-usage)
+    - [Requirements](#requirements)
```
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md [hunk 5]
```diff
@@ -1,6 +1,19 @@
 # KNoT Babeltower
 
-## Installation and usage
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/9140aa8c06934071ad6e3cf3b1b148ff)](https://www.codacy.com/manual/joaoaneto/knot-babeltower?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=CESARBR/knot-babeltower&amp;utm_campaign=Badge_Grade)
+
+## Contents
+ - [Basic installation and usage](#basic-installation-and-usage)
```
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md [hunk 6]
```diff
@@ -1,6 +1,19 @@
...
+    - [Requirements](#requirements)
+    - [Configuration](#configuration)
+    - [Setup](#setup)
+    - [Compiling and Running](#compiling-and-running)
+ - [Docker installation and usage](#docker-installation-and-usage)
```
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 2 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 4 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md [hunk 7]
```diff
@@ -1,6 +1,19 @@
 # KNoT Babeltower
 
-## Installation and usage
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/9140aa8c06934071ad6e3cf3b1b148ff)](https://www.codacy.com/manual/joaoaneto/knot-babeltower?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=CESARBR/knot-babeltower&amp;utm_campaign=Badge_Grade)
+
+## Contents
+ - [Basic installation and usage](#basic-installation-and-usage)
+    - [Requirements](#requirements)
+    - [Configuration](#configuration)
+    - [Setup](#setup)
```
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md [hunk 8]
```diff
@@ -33,9 +46,13 @@ make deps
 make run
 ```
 
-## Local (Development)
+## Docker installation and usage
+
+### Requirements
+
+*   Install docker engine (<https://docs.docker.com/install/>)
```
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[unordered-list-marker-style] Marker style should be `-`](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md [hunk 9]
```diff
@@ -1,6 +1,19 @@
 # KNoT Babeltower
 
-## Installation and usage
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/9140aa8c06934071ad6e3cf3b1b148ff)](https://www.codacy.com/manual/joaoaneto/knot-babeltower?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=CESARBR/knot-babeltower&amp;utm_campaign=Badge_Grade)
+
+## Contents
+ - [Basic installation and usage](#basic-installation-and-usage)
+    - [Requirements](#requirements)
+    - [Configuration](#configuration)
```
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md [hunk 10]
```diff
@@ -33,9 +46,13 @@ make deps
 make run
 ```
 
-## Local (Development)
+## Docker installation and usage
+
+### Requirements
```
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[no-duplicate-headings] Do not use headings with similar content (18:1)](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 5 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 6 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 9 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 7 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 3 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 1 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

**Comment by joaoaneto** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 20
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4651373&bid=15577405)

File: README.md (hunk 5 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 2 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 1 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 7 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 3 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 2 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 7 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 4 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 5 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 9 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 4 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 9 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 6 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md [hunk 11]
```diff
@@ -33,9 +46,15 @@ make deps
 make run
 ```
 
-## Local (Development)
+> You can use the `make watch` command to run the application on watching mode, allowing it to be restarted automatically when the code changes.
+
+## Docker installation and usage
+
+### Requirements
```
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[no-duplicate-headings] Do not use headings with similar content (18:1)](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 1 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 2 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 6 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md [hunk 12]
```diff
@@ -33,9 +46,15 @@ make deps
...
+
+*   Install docker engine (<https://docs.docker.com/install/>)
```
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[unordered-list-marker-style] Marker style should be `-`](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 3 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

**Comment by joaoaneto** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 20
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4651373&bid=15577405)

File: README.md (hunk 2 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 11 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[no-duplicate-headings] Do not use headings with similar content (18:1)](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 4 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 3 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 4 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 2 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 6 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 12 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[unordered-list-marker-style] Marker style should be `-`](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 1 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 9 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 6 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 2 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 5 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 3 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 1 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 5 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 1 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 7 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 2 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 11 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[no-duplicate-headings] Do not use headings with similar content (18:1)](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 4 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 3 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 4 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 2 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 6 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 12 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[unordered-list-marker-style] Marker style should be `-`](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 1 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 9 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 6 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 2 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 5 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 3 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 1 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 5 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 1 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

File: README.md (hunk 7 above)
Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

Comment (joaoaneto) [inline README.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4651373)

**Review by vitbaq — APPROVED** [review]

**Pull request merged**